### PR TITLE
feature--main-column-tabs

### PR DESCRIPTION
**Pull request by jeffworley**

### CODE REVIEW

**Comment by jeremyrist** [comment]
:+1: solid as a rock, those octicons going crazy

**Comment by s4dmw** [comment]
dang thats a lotta files bruh. other than that it looks on point :+1:

File: src/bower_components/octicons/.bower.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
```
Comment (kellymurray) [inline src/bower_components/octicons/.bower.json:1]: Don't commit your `bower_components` - think of them like the `node_modules`.

File: src/index.html [hunk 2]
```diff
@@ -29,24 +30,66 @@
...
+              </nav>
+            </section>
+            <section id="repos">
+              <ul id="repo-list">
+                <li>Repo 1</li>
+                <li>Repo 2</li>
+                <li>Repo 3</li>
+                <li>Repo 4</li>
+              </ul>
+            </section>
```
Comment (kellymurray) [inline src/index.html:68]: `navs` within `sections` within -- ! did the example in class today help you think of ways to make this DRY code?

**Pull request merged**